### PR TITLE
Express dezuciu hover  #11

### PR DESCRIPTION
**Pull request by vitgri1**

### CODE REVIEW

**Review by Arshunas — DISMISSED** [review]

File: css/pages/home.css [hunk 1]
```diff
@@ -118,6 +118,7 @@
     border-radius: 1em;
     box-shadow: 0 0.25em 0.5em rgba(0,0,0,.075);
```
Comment (xzcjdvd) [inline css/pages/home.css:119]: Noretusi man asmeniskai gal bent didesniu seseliu pagal dizaina?

**Review by xzcjdvd — CHANGES_REQUESTED** [review]
Viskas super apart seseli

**Review by inga911 — APPROVED** [review]
LGTM! 👍

**Review by xzcjdvd — APPROVED** [review]
Super!

**Pull request merged**